racket/collects/racket/private/logger.rkt

```diff
@@ -27,15 +27,17 @@
   (define-syntax log-info (make-define-log 'info #'(current-logger) #'(logger-name l)))
   (define-syntax log-debug (make-define-log 'debug #'(current-logger) #'(logger-name l)))
 
-  (define (check-logger who)
-    (lambda (v)
-      (unless (logger? v)
-        (raise-argument-error who "logger?" v))
-      v))
+  (define (check-logger who v)
```

Comment:
Were there no uses of `check-logger` elsewhere?

Reply:
Not as far as I can tell.